lib/IRGen/GenKeyPath.cpp

```diff
@@ -188,22 +188,22 @@ getAccessorForComputedComponent(IRGenModule &IGM,
   case Getter:
     // Original accessor's args should be @in or @out, meaning they won't be
     // captured or aliased.
-    accessorThunk->addAttribute(1, llvm::Attribute::NoCapture);
```

Comment:
@jckarter I'm fairly sure this is correct, but if you want to double check that'd be great :). `addParamAttr` eventually adds `FirstArgIndex`, which is 1.